src/server/repository/assessment/assessment/createAssessment.ts

```diff
@@ -7,15 +7,17 @@ import { read } from '@server/repository/assessment/assessment/read'
 
 export const createAssessment = async (
   params: {
-    assessment: Pick<Assessment, 'props'>
+    assessment: Pick<Assessment, 'props' | 'metaCache'>
```

Comment:
when creating an assessment, metaCache is not be populated. 
can u avoid passing metaCache here?

Reply:
how should we populate the test assessment?
we fetch it from db and if we dont add it there, dont add it here, then.. what is left?

Reply:
check my pr